nvflare/lighter/poc.py

```diff
@@ -32,11 +32,16 @@ def clone_client(num_clients: int, dest_poc_folder):
     shutil.rmtree(src_folder)
 
 
-def unpack_poc() -> bool:
+def unpack_poc(dest_poc_folder) -> bool:
     file_dir_path = pathlib.Path(__file__).parent.absolute()
     poc_zip_path = file_dir_path.parent / "poc.zip"
     try:
-        shutil.unpack_archive(poc_zip_path)
+        import tempfile
+
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            shutil.unpack_archive(poc_zip_path, extract_dir=tmp_dir.name)
```

Comment:
tmp_dir is the name of the temp directory, so it's a string.  I think you don't need tmp_dir.name.

Reply:
```
$ python3
Python 3.10.4 (main, Jun 29 2022, 12:14:53) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tempfile
>>> with tempfile.TemporaryDirectory() as tmp_dir:
...   print(tmp_dir)
...   print(tmp_dir.name)
... 
/tmp/tmpyk0leofm
Traceback (most recent call last):
  File "<stdin>", line 3, in <module>
AttributeError: 'str' object has no attribute 'name'
```